### DISCUSSION TITLE
Meeting: Celo Wallet and (d)App Council #14 [August] 💬

### DISCUSSION THREAD

**Discussion by arthurgousset** [celo-org/celo-monorepo#9695] [Wallet and (d)App Council]
# Monthly Celo Wallet and (d)App Council

##  📆 Quick links 

- When: **Wednesday, August 17 (9am PDT/6pm CEST)**
- Where: **[Google Meet](https://meet.google.com/uwj-wfse-uzm)**
- Agenda: **comments**👇
- Previous: https://github.com/celo-org/celo-monorepo/discussions/9662
- Next: https://github.com/celo-org/celo-monorepo/discussions/9774
- Recording: [Google Drive](https://drive.google.com/file/d/1WKpL3_bzicDY2k9NrzQeOqfGiMIsDLC7/view?usp=sharing)

## 🤷‍♂️ What is this?

A monthly meeting for Celo **developers, founders,** **product** people (anyone really!) to come together and
- find consensus (coordinate),
- learn about needs, pain points, and desires
- present and collect technical feedback

## 🗒️  Agenda

- Comment with a short sentence describing what you'd like to discuss
- Don't worry, it can be small or big! 
- Every builder is welcome (and we are very easy going, I promise 😄)
- Upvote topics you're interested in

## Housekeeping

_You can safely ignore this!_

- [x] Send update calendar invite with Github link
- [x] Encourage additions to the agenda
- [x] Publicise on forum and Discord
- [x] RECORD meeting 🎥
- [ ] Share recording below and on forum
- [ ] Create next meeting (link at `next` above)

**Comment by arthurgousset** [comment]
@product-matt kindly agreed to facilitate this meeting while I'm OOO 🙌 I'll be back for the next meeting 🏖️ 
_Don't forget to click record_ :) 🎥

**Comment by fbaud** [comment]
While this is still very early, we will see more and more requirements in new regulations (e.g. the European MiCA regulation) to handle identity when an unhosted wallet is interacting with an exchange. More generally, we will probably need to see DApps requesting some sort of proof of ID when interacting with a Wallet in specific circumstances (e.g. transferring amounts above certain thresholds,...).
Maybe we can spend a couple of minutes starting exchanging on the theme just to get the ball rolling for the discussions in the upcoming months.

**Reply by product-matt** [reply]
agreed very important and interesting topic! Maybe someone from the mento team can join, I'm sure regulation is very top of mind for them.

**Comment by product-matt** [comment]
Launch of StakedCelo

**Comment by aaronmgdr** [comment]
React Celo. 4.1 release planned for early next week 

Special thanks to the Mobius team for trying out the beta and providing crucial feedback and bug fixes

**Comment by aaronmgdr** [comment]
React Celo Network Handling Discussion https://github.com/celo-org/react-celo/discussions/297

**Comment by arthurgousset** [comment]
📽 Here is the [recording](https://drive.google.com/file/d/1WKpL3_bzicDY2k9NrzQeOqfGiMIsDLC7/view?usp=sharing) for anyone that missed the meeting.
